packages/eslint-plugin-orbit-internal/package.json

```diff
@@ -8,7 +8,8 @@
     "postinstall": "yarn build"
   },
   "dependencies": {
-    "@babel/types": "=7.12.10"
+    "@babel/types": "=7.12.10",
+    "eslint-plugin-orbit-components": "^0.3.0"
```

Comment:
@silvenon I would like to have it also used internally, but currently we don't build the `eslint-plugin-orbit-components` locally for development purposes, but it would be required. Other option would be to have duplicate implementation, or to use publish `@kiwicom/eslint-plugin-orbit-internal` and have it use it from there.

Reply:
I vote for building `eslint-plugin-orbit-components` on `postinstall`; we'll have more and more rules that we'll want to use in our own codebase, and compiling the plugin is fast. No need to re-export it from `eslint-plugin-orbit-internal`.

Assuming you agree, when implementing building on `postinstall` don't forget to run `pinst` on `prepublishOnly` and `postpublish` just like in `@kiwicom/orbit-components`. Also, in root `.eslintrc.js` you can add `plugin:orbit-components/recommended` to `extends`. And while you're at it, you could also add `plugin:@kiwicom/orbit-internal/recommended`, too, I guess that works now (it didn't work for me before), and remove its rules and remove it from `plugins` because that's all included in the recommended configuration.

Reply:
I vote for building `eslint-plugin-orbit-components` on `postinstall`; we'll have more and more rules that we'll want to use in our own codebase, and compiling the plugin is fast. No need to re-export it from `eslint-plugin-orbit-internal`.

Assuming you agree, when implementing building on `postinstall` don't forget to run `pinst` on `prepublishOnly` and `postpublish` just like in `@kiwicom/orbit-components`. Also, in root `.eslintrc.js` you can add `plugin:orbit-components/recommended` to `extends`. And while you're at it, you could also add `plugin:@kiwicom/orbit-internal/recommended`, too, I guess that works now (it didn't work for me before), and remove its rules and remove it from `plugins` because that's all included in the recommended configuration.

Reply:
I will create separate PR for this, if you agree. This PR will be merged into `dev/tokens`, but we might benefit from it on master already and the scope would be outside of this PR.

Reply:
I will add the rule just into `eslint-plugin-orbit-components` for now. We don't have to bother we the build cycle currently.